### PR TITLE
fix data preprocess when predicting in duee

### PR DESCRIPTION
**Pull request by Steffy-zxf**
### PR types

 Bug fixes

### PR changes
 Others

### Describe

fix data preprocess when predicting in duee

### CODE REVIEW

**Review by ZeyuChen — APPROVED** [review]
LGTM

File: paddlenlp/datasets/experimental/chnsenticorp.py [hunk 1]
```diff
@@ -48,13 +48,12 @@ class ChnSentiCorp(DatasetBuilder):
 
     def _get_data(self, mode, **kwargs):
         """Downloads dataset."""
-        default_root = DATA_HOME
+        default_root = os.path.join(DATA_HOME, "ChnSentiCorp")
```
Comment (smallv0221) [inline paddlenlp/datasets/experimental/chnsenticorp.py]: default_root = os.path.join(DATA_HOME, self.__class__.__name__)

File: paddlenlp/datasets/experimental/lcqmc.py [hunk 2]
```diff
@@ -47,13 +47,12 @@ class LCQMC(DatasetBuilder):
     }
 
     def _get_data(self, mode, **kwargs):
-        default_root = DATA_HOME
+        default_root = os.path.join(DATA_HOME, "LCQMC")
```
Comment (smallv0221) [inline paddlenlp/datasets/experimental/lcqmc.py]: 同上

File: paddlenlp/datasets/experimental/chnsenticorp.py (hunk 1 above)
Comment (Steffy-zxf) [reply paddlenlp/datasets/experimental/chnsenticorp.py]: done

File: paddlenlp/datasets/experimental/lcqmc.py (hunk 2 above)
Comment (Steffy-zxf) [reply paddlenlp/datasets/experimental/lcqmc.py]: done

**Review by smallv0221 — APPROVED** [review]
LGTM

**Pull request closed without merging**